src/core/ext/xds/xds_cluster.h

```diff
@@ -101,8 +101,8 @@ struct XdsClusterResource : public XdsResourceType::ResourceData {
 
   XdsHealthStatusSet override_host_statuses;
 
-  RefCountedStringValue service_telemetry_label;
-  RefCountedStringValue namespace_telemetry_label;
+  RefCountedStringValue service_telemetry_label_;
```

Comment:
These fields should *not* have an underscore suffix, since this is a struct, not a class.